src/data/ff/atomtype.h

```diff
@@ -3,25 +3,19 @@
 
 #pragma once
 
-#include "base/parameters.h"
 #include "data/elements.h"
 #include "neta/neta.h"
-#include "templates/optionalref.h"
 
 // Forward Declarations
 class Forcefield;
-class ForcefieldParameters;
 
 // Forcefield AtomType Base Class
 class ForcefieldAtomType
 {
     public:
     ForcefieldAtomType(Elements::Element Z = Elements::Unknown, int index = -1, std::string_view name = "",
                        std::string_view netaDefinition = "", std::string_view description = "", double q = 0.0,
-                       double data0 = 0.0, double data1 = 0.0, double data2 = 0.0, double data3 = 0.0);
-    ForcefieldAtomType(OptionalReferenceWrapper<const ForcefieldParameters> params, Elements::Element Z = Elements::Unknown,
-                       int index = -1, std::string_view name = "", std::string_view netaDefinition = "",
-                       std::string_view description = nullptr, double q = 0.0);
+                       const std::vector<double> parameters = {}, std::string_view equivalentName = "");
```

Comment:
```suggestion
                       const std::vector<double> &parameters = {}, std::string_view equivalentName = "");
```

Passing by reference to prevent copying.  Default values for const reference parameters should be allowed.